src/test/java/net/openhft/chronicle/bytes/readme/PrimitiveTest.java

```diff
@@ -221,9 +221,22 @@ public void testBinaryPrimitiveOffset() {
     }
 
     @Test
-    public void testTextPrimitive() {
+    public void testTextPrimitiveByteBuffer() {
+        doTestTextPrimitive(Bytes.elasticHeapByteBuffer(64));
```

Comment:
these tests do not break if I revert the change HeapBytesStore

Reply:
They do, but I still think more code coverage do not hurt. They used to be broken by something else which since have been taken care of, so I believe.

Reply:
Fair enough!